app/uk/gov/hmrc/thirdpartyapplication/services/ApplicationService.scala

```diff
@@ -229,7 +182,7 @@ class ApplicationService @Inject()(applicationRepository: ApplicationRepository,
       app <- fetchApp(applicationId)
       _ <- deleteSubscriptions(app)
       _ <- thirdPartyDelegatedAuthorityConnector.revokeApplicationAuthorities(app.tokens.production.clientId)
-      _ <- apiGatewayStore.deleteApplication(app.wso2Username, app.wso2Password, app.wso2ApplicationName)
+      _ <- apiGatewayStore.deleteApplication(app.wso2ApplicationName)
```

Comment:
Naming ?

Reply:
Nope - this is the one exception in the refactor. `wso2ApplicationName` is the name of the field in MongoDB, and is used for API Keys in AWS.